### DISCUSSION TITLE
How to use Observable code and "this" in Quarto ojs cell?

### DISCUSSION THREAD

**Discussion by Juuso1** [quarto-dev/quarto-cli#10032] [Q&A] [answered]
### Description

I try to copy this code (https://observablehq.com/@fil/whats-this) to Quarto document, but I can´t see nothing. Only Replay button an this one "Object {value: NaN}" ? What should I do?

```{ojs}
viewof replay = html`<button>replay`
```
```{ojs}
{
  replay;
  mutable that = this;
  if (this) yield { value: this.value + 1 };
  else yield { value: 0 };
}
```
```{ojs}
mutable that = undefined
```

Br, Juuso

**Comment by cscheid** [comment, marked as answer]
I think this is a bug. Tracking on #10035.
